server/ntb/tests/io/feed_parsers/dpa_newsml_test.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8; -*-
+#
+# This file is part of Superdesk.
+#
+# Copyright 2013 - 2018 Sourcefabric z.u. and contributors.
+#
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+
+
+from ntb.io.feed_parsers.dpa_newsml import DPANewsMLFeedParser
+from . import XMLParserTestCase
+
+
+class BaseDPANewsMLTestCase(XMLParserTestCase):
+    def setUp(self):
+        self.parser = DPANewsMLFeedParser()
+        super().setUp()
+
+
+class DPATestCase(BaseDPANewsMLTestCase):
+    filename = 'dpa_newsml_test.xml'
+
+    def test_can_parse(self):
+        self.assertTrue(DPANewsMLFeedParser().can_parse(self.xml_root))
+
+    def test_content(self):
+        item = self.item[0]
+        self.assertEqual(item['subject'], [{'qcode': 'Utenriks', 'name': 'Utenriks', 'scheme': 'category'}])
+        self.assertEqual(item['anpa_category'], [{'name': 'Nyhetstjenesten', 'qcode': 'n'}])
+        self.assertEqual(item['headline'], 'Chef von Ölkonzern Eni soll wegen Nigeria-Schmiergeldern vor Gericht')
+        self.assertEqual(item['guid'], 'urn:newsml:dpa.com:20090101:171220-99-357156:6')
+        self.assertEqual(item['uri'], 'urn:newsml:dpa.com:20090101:171220-99-357156')
+        self.assertEqual(item['firstcreated'].isoformat(), '2017-12-20T14:49:36+01:00')
+
+        expected_body = ('<div><header>  <ul class="slugline">    <li class="subject">Energie</li>\n    <li class="subj'
```

Comment:
imo there should be no `header` or `section` in the body

Reply:
I was not sure if I should keep the HTML as it is or not. The issue is that HTML can have many elements, so we should maybe have a list of elements we can keep or not? And change elements not whitelisted to <div> for instance?